crates/autopilot/src/solvable_orders.rs

```diff
@@ -234,7 +234,8 @@ impl SolvableOrdersCache {
         counter.record(&auction.orders);
 
         if self.store_in_db {
-            let _id = self.database.replace_current_auction(&auction).await?;
+            let id = self.database.replace_current_auction(&auction).await?;
+            tracing::info!(auction = %id, %block, "new auction");
```

Comment:
Without reading up on the code where the log is produced I would not have known that `new auction` only refers to the DB.
Maybe `stored new auction in DB` and `stored new auction in autopilot cache` would be slightly more helpful.